cmd/download.go

```diff
@@ -145,3 +155,11 @@ func removeDir(dir string) error {
 	}
 	return nil
 }
+
+func renameDir(src string, dest string) error {
+	err := os.Rename(src, dest)
```

Comment:
maybe would be nice to check here if `src` and `dest` are folders here before renaming?